shuup/core/models/_orders.py

```diff
@@ -276,13 +333,47 @@ def ensure_default_statuses(self):
             else:
                 OrderStatus.objects.create(**defaults)
 
+        self.ensure_allowed_next_statuses()
+
+    def ensure_allowed_next_statuses(self):
+        """
+        Populates allowed_next_statuses with default values.
+        """
+        # run only if there are no allowed_next_statues defined.
+        os_qs = OrderStatus.objects.filter(~Q(allowed_next_statuses=None))
+        if not os_qs:
+            order_status_qs = OrderStatus.objects.all()
+            for order_status in order_status_qs:
```

Comment:
ok, there is no way for us to really decide if we can actually run this `ensure_allowed_next_statuses`.
Let's go back to the migration idea, so it will run only once.
Create a migration in core that will call this and that's it.